views/agency-requests/view_participant.php

```diff
@@ -298,21 +301,22 @@ function checkSet($value){
 						<th>Sex</th>
 					</tr>			
 					<?php
-					#if query returns nothing, throw an error to the user
-					if(pg_num_rows($familyInfo) == 0){
-						echo "no family found";
-					}else{
-						while ($rowF = pg_fetch_assoc($familyInfo)) {
-							echo 	"<tr>".
-									"<td>".checkSet($rowF['lastname'])." ".$rowF['middleinit']." ".checkSet($rowF['lastname'])."</td>".
-									"<td>".checkSet($rowF['relationship'])."</td>".
-									"<td>".checkSet($rowF['dateofbirth'])."</td>".
-									"<td>".checkSet($rowF['sex'])."</td>".
-									"</tr>";
+                    // Display all family information
+                    $familyRow = pg_fetch_all($familyInfo);
+                    if(pg_num_rows($familyInfo) > 0){
+                    foreach($familyRow as $key=>$value){
```

Comment:
Instead of $key=>$value could you just do 

```php
foreach ($familyRow as $row) { 
```
I don't think you use key anywhere?